store/postgres/migrations/000021_add_job_deployment_table.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+CREATE TABLE IF NOT EXISTS job_deployment (
+    id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    project_id UUID NOT NULL REFERENCES project (id),
+    status VARCHAR(15) NOT NULL,
+    details JSONB,
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    updated_at TIMESTAMP WITH TIME ZONE
```

Comment:
Are we expecting this field to be nullable ?

Reply:
for the `updated_at`, yes, it can be null if it is just created.

Reply:
Was wondering as gorm has this field marked as not null
```
UpdatedAt time.Time `gorm:"not null" json:"updated_at"`
```

Reply:
oh just realized it. I can remove the `not null` from gorm.

Reply:
actually seeing from other tables, we are keeping UpdatedAt as NOT NULL, I think the value is filled with when the record is created initially. maybe I can change to follow this (having it as not null), what do you think? @sbchaos

Reply:
Sure, let's keep the field as not null as in other tables.

Reply:
updated this